tests/management/access/test_view.py

```diff
@@ -138,3 +140,66 @@ def test_missing_invalid_username(self):
         client = APIClient()
         response = client.get(url, **self.headers)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_admin_RonR(self):
+        """Test that a admin user can access RBAC resources"""
+        url = '{}?application={}&username={}'.format(reverse('access'), 'rbac', self.principal.username)
```

Comment:
So, this is checking against the access endpoint, correct? Like `/access/?application=rbac`? Shouldn't that always return a 200, just with an empty dataset, whereas actual RBAC resources (groups/roles, etc) would be limited by this change? I may be missing something, though. 🤔

Reply:
I went back and forth on which endpoint to test, ended up hitting access primarily to have a decision made and something to review. I honestly wasn't sure how access was to work after this change with a non-admin user, but that may not be the case based on your testing below. I think for ease of use and understanding I'll swap these tests to a different endpoint and then maybe step through execution to determine what's happening on the non-adming access test